src/compiler/scanner.ts

```diff
@@ -1508,9 +1512,9 @@ namespace ts {
         }
 
         function getIdentifierToken(): SyntaxKind.Identifier | KeywordSyntaxKind {
-            // Reserved words are between 2 and 11 characters long and start with a lowercase letter
+            // Reserved words are between 2 and 12 characters long and start with a lowercase letter
             const len = tokenValue.length;
-            if (len >= 2 && len <= 11) {
+            if (len >= 2 && len <= 12) {

```

Comment:
Note that this change should be reverted since #40580 has removed `uncapitalize` and `intrinsic` is 9 characters long.